regolith/helpers/l_contactshelper.py

```diff
@@ -87,28 +94,32 @@ def construct_global_ctx(self):
 
     def sout(self):
         rc = self.rc
+        if rc.find:
+            results = search_collection(self.gtx["contacts"], rc.find)
```

Comment:
maybe the function should be called ``key_value_pair_filter`` to be clearer?

then it makes most sense that it takes a list of doubles (i.e., key-value pairs) or sthg like that.  

You can build the right data structure from the argparser here, not in the function (and validated that there are an even number of args)